doc/quick_start/getting_started_with_Dashing_Ubuntu18.04_ov2.0.md

```diff
@@ -0,0 +1,168 @@
+# ROS2_OpenVINO_Toolkit
+
+**NOTE:** 
+Below steps have been tested on **Ubuntu 18.04**.
+
+## 1. Environment Setup
+* Install ROS2 Dashing ([(guide)](https://docs.ros.org/en/dashing/Installation/Ubuntu-Install-Debians.html))
+* Install Intel® OpenVINO™ Toolkit Version: 2022.1 ([guide](https://docs.openvino.ai/2022.1/openvino_docs_install_guides_installing_openvino_linux.html))
+  * Install from an achive file ([guide](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/download.html))
+  Tips: Both runtime and development tool are needed, `pip` is recommended for installing the development tool ([guide](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html))
+  * Install from source code ([guide](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux))
+* Install Intel® RealSense™ SDK ([guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md))
+
+## 2. Building and Installation
+* Install ROS2_OpenVINO packages
+```
+mkdir -p ~/catkin_ws/src
+cd ~/catkin_ws/src
+git clone https://github.com/intel/ros2_openvino_toolkit -b dashing
```

Comment:
<img width="738" alt="image" src="https://user-images.githubusercontent.com/92708152/202972743-281fca01-50c3-43cb-94f4-86ebcbdcf79b.png">
there have the 5 numbers to git clone, but 4 numbers git clone in the dockerfile

Reply:
Some package could be removed.